### ISSUE TITLE
Version number of "1.4" produces semver warning

### ISSUE THREAD

**Issue by esfalsa** [Vleerian/FattKATT#11]
### Subject of the issue
The current version number seems to be [1.4](https://github.com/Vleerian/FattKATT/blob/b1b571e214933c0f04543ac187a353d6e59abeb2/Program.cs#L39), which produces a warning when running FattKATT about invalid semantic versioning:

```
FatKATT Version 1.4.
This software is provided as-is, without warranty of any kind.
FatKATT [REQ] - Checking for newer versions...
FatKATT [WRN] - Invalid semantic versioning - it is recommended to use the latest official 
release.
```

### Your environment
* FattKATT 1.4
* macOS 13.1

### Steps to reproduce
1. Run FattKATT
2. That's it 😛 

### Expected behaviour
I assume the version is probably meant to be 1.4.0.

### Actual behaviour
The version number of 1.4 produces a warning.

**Comment by Vleerian** [comment]
Yep, I'll fix the version number shortly

**Comment by Vleerian** [comment]
Addressed in 678591c
